<launch>

  <node pkg="arkit_streamer" type="pose_server.py" name="pose_server" output="screen">
   <param name="port_number" value="35601" type="int"/>
   <param name="pose_topic" value="ios_pose"/>
   <param name="coordinate_frame" value="odom"/>
  </node>

  <node pkg="arkit_streamer" type="image_server.py" name="image_server" output="screen">
   <param name="port_number" value="35602" type="int"/>
   <param name="camera_name" value="camera"/>
 </node>

 <node pkg="arkit_streamer" type="aprilTag_server.py" name="aprilTag_server" output="screen">
  <param name="port_number" value="35603" type="int"/>
</node>

<node pkg="arkit_streamer" type="phone_type_server.py" name="phone_type_server" output="screen">
</node>

<!-- <node pkg="apriltags_ros" type="apriltag_detector_node" name="apriltag_detector" output="screen" ns="camera_lowres"> -->
  <!-- Remap topic required by the node to custom topics -->
  <!-- <remap from="image_rect" to="image_raw" />
   <remap from="camera_info" to="camera_info" /> -->
  <!-- Optional: Subscribe to the compressed stream-->
  <!-- <param name="image_transport" type="str" value="compressed" /> -->

  <!-- Select the tag family: 16h5, 25h7, 25h9, 36h9, or 36h11(default) -->
  <!-- <param name="tag_family" type="str" value="36h11" /> -->

  <!-- Enable projected optical measurements for more accurate tag transformations -->
  <!-- This exists for backwards compatability and should be left true for new setups -->
  <!-- <param name="projected_optics" type="bool" value="true" /> -->

  <!-- Describe the tags -->

  <!-- <rosparam param="tag_descriptions">

    [{id: 0, size: 0.165},
    {id: 1, size: 0.165},
    {id: 2, size: 0.165},
    {id: 3, size: 0.165},
    {id: 4, size: 0.165},
    {id: 5, size: 0.165},
    {id: 6, size: 0.165},
    {id: 7, size: 0.165},
    {id: 8, size: 0.165},
    {id: 9, size: 0.165},
    {id: 10, size: 0.165},
    {id: 11, size: 0.165},
    {id: 12, size: 0.165},
    {id: 13, size: 0.053},
    {id: 14, size: 0.165},
    {id: 15, size: 0.165},
    {id: 16, size: 0.165},
    {id: 17, size: 0.165},
    {id: 18, size: 0.165},
    {id: 19, size: 0.165},
    {id: 20, size: 0.165},
    {id: 21, size: 0.165},
    {id: 22, size: 0.165},
    {id: 23, size: 0.165},
    {id: 24, size: 0.165},
    {id: 25, size: 0.165},
    {id: 26, size: 0.165},
    {id: 27, size: 0.165},
    {id: 28, size: 0.165},
    {id: 29, size: 0.165},
    {id: 30, size: 0.165},
    {id: 31, size: 0.165},
    {id: 32, size: 0.165},
    {id: 33, size: 0.165},
    {id: 34, size: 0.165},
    {id: 35, size: 0.165},
    {id: 36, size: 0.165},
    {id: 37, size: 0.165},
    {id: 38, size: 0.165},
    {id: 39, size: 0.165}]
  </rosparam>
</node> -->

<node pkg="image_transport" type="republish" name="decompress_camera" args="in:=camera/image_raw compressed out:=camera/image_raw raw"/>
<node pkg="image_transport" type="republish" name="decompress_camera_lowres" args="in:=camera/image_lowres compressed out:=camera/image_lowres raw"/>

</launch>
